pkg/api/server/register_images.go

```diff
@@ -762,6 +762,29 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: '#/responses/InternalError'
 	r.Handle(VersionedPath("/libpod/images/{name:.*}/exists"), s.APIHandler(libpod.ImageExists)).Methods(http.MethodGet)
+	// swagger:operation GET /libpod/images/{name}/resolve libpod ImageResolveLibpod
+	// ---
+	// tags:
+	//  - images
+	// summary: Resolve an image name to one more fully-qualified names
+	// description: Allows for resolving short-names to one more fully-qualified names, depending on the server configuration
+	// parameters:
+	//  - in: path
+	//    name: name
+	//    type: string
+	//    required: true
+	//    description: the name or ID of the image
+	//  - in: query
+	//    name: policy
+	//    type: boolean
```

Comment:
Type is boolean and the description enumerates strings. Also, if parameter is required then you don't really have a default.